### PR TITLE
feat: Move new ImageFeaturizer to onnx namespace

### PR DESCRIPTION
**Pull request by svotaw**
## Related Issues/PRs

#xxx

## What changes are proposed in this pull request?

Move the ImageFeaturizer from the cntk to the onnx namespace, and clean up some tests

## How is this patch tested?

PR pipeline

## Does this PR change any dependencies?

- [x ] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

1. Find the corresponding markdown file for your new feature in `website/docs/documentation` folder.
   Make sure you choose the correct class `estimators/transformers` and namespace.
2. Follow the pattern in markdown file and add another section for your new API, including pyspark, scala (and .NET potentially) samples.
3. Make sure the `DocTable` points to correct API link.
4. Navigate to website folder, and run `yarn run start` to make sure the website renders correctly.
5. Don't forget to add `<!--pytest-codeblocks:cont-->` before each python code blocks to enable auto-tests for python samples.
6. Make sure the `WebsiteSamplesTests` job pass in the pipeline.

### CODE REVIEW

File: deep-learning/src/main/scala/com/microsoft/azure/synapse/ml/onnx/ImageFeaturizer.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+The ``ImageFeaturizer`` relies on a ``ONNXModel`` to do the
+featurization of the image(s).  One can set this model directly using the
+onnxModel parameter, or use setModel to set a known model name that will
```
Comment (mhamilton723) [inline deep-learning/src/main/scala/com/microsoft/azure/synapse/ml/onnx/ImageFeaturizer.txt:3]: these text files are no longer part of our build. Perhaps we can just incorporate into scala docstring if needed

**Review by mhamilton723 — APPROVED** [review]
Approved, but feel free to get rid of the .txt and move it instead to some scaladoc

File: deep-learning/src/main/scala/com/microsoft/azure/synapse/ml/onnx/ImageFeaturizer.txt (hunk 1 above)
Comment (svotaw) [reply deep-learning/src/main/scala/com/microsoft/azure/synapse/ml/onnx/ImageFeaturizer.txt:3]: ok, I have a separate task to cleanup related docs and add examples, so I'll do that as part of that task.

**Review by mhamilton723 — APPROVED** [review]

**Pull request merged**